assets/generation/pip/requirements.txt

```diff
@@ -8,8 +8,8 @@
 # Update me and then run 'docker-compose -f dcp-generation.yml run assets pip_freeze'	#
 #                                                                                       #
 #########################################################################################
-aether.client==0.0.0
-aether.mocker==0.0.0
+aether.client==0.10.0
```

Comment:
Can we get these versions in sync, i.e. 0.11.0?

Reply:
These are manually included in the generation application. It's requirements aren't necessarily fixed to the current version of Aether.

Reply:
There isn't a `v.11` of `mocker` as it's not automatically built.

Reply:
Update: The mocker script has been moved directly to bootstrap ( as it was only used in the generation function )